content/docs/card/email-feature-card.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Email Feature Card
+description: A card component that display From, to, Subject and  Body of the email and a send icon with animation on appearance and hover over it.
+author: raghav_dadhich
+---
+
+<ComponentPreview name="card-email-feature-card--docs" />
+
+## Installation
+
+<Steps>
+<Step>Install dependencies</Step>
+
+```bash
+npm install framer-motion lucide-react
+```
+
+<Step>Update `tailwind.config.js`</Step>
+
+Add the following to your tailwind.config.js file.
+
+```json
+module.exports = {
+  theme: {
+    extend: {
+    }
+  }
+}
+```
+
```

Comment:
You don't need these two steps, you can delete them @raghav3615

Reply:
Done